### ISSUE TITLE
User Story: **Complete Model Training**

### ISSUE THREAD

**Issue by whelew** [whelew/P-Five-Mildew#13]
As a **Data Engineer** I can **have a fully trained model** so that **it can successfully predict to a 97% accuracy whether mildew dust is present within cherry leaves**

**Acceptance Criteria:**

- Test the test set using the most optimal model.

- If accuracy of tracked performance metric is 97% the model is trained to an acceptable level.

- If accuracy of tracked performance metric is below 97%, retrain model using new hyperparameters. 

- Repeat the previous processes of model optimisation
